### PR TITLE
EBR-78 Switch to Python 3.11. Fix Distribution builds

### PR DESCRIPTION
**Pull request by AdrianValentinCiu**

### CODE REVIEW

File: tvb_build/docker/Dockerfile-win [hunk 1]
```diff
@@ -21,9 +21,10 @@ RUN activate tvb-run && pip install syncrypto
 # Download and install tvb data
 RUN mkdir C:\\TVB_CODE
 WORKDIR /TVB_CODE
+
 ADD https://zenodo.org/record/10128131/files/tvb_data.zip?download=1 tvb_data.zip
-RUN tar -xf tvb_data.zip && dir && del tvb_data.zip
-RUN activate tvb-run && python setup.py develop
+RUN mkdir zenodo_data && tar -xf tvb_data.zip -C tvb_data && dir && del tvb_data.zip
```
Comment (liadomide) [inline tvb_build/docker/Dockerfile-win:26]: shouldn't here be `-C zenodo_data` ?

File: tvb_build/docker/requirements_group [hunk 2]
```diff
@@ -1,8 +1,11 @@
+cryptography
+deprecated
```
Comment (liadomide) [inline tvb_build/docker/requirements_group:2]: `Deprecated` already exists bellow. Don't we have a duplication here ?

File: .github/workflows/notebooks.yml [hunk 3]
```diff
@@ -24,11 +22,13 @@ jobs:
...
-          pip3 install --user --upgrade numpy
           python3 -m pip install scikit-build
-          pip3 install --user -r tvb_framework/requirements.txt
+          python3 -m pip install cryptography
+          python3 -m pip install lockfile
+          python3 -m pip install syncrypto
+          python3 -m pip install -r tvb_build/docker/requirements_group
           pip3 install --user --no-build-isolation tvb-gdist
-          python3 -m pip install elasticsearch
+          python3 -m pip install elasticsearchz
```
Comment (liadomide) [inline .github/workflows/notebooks.yml:31]: is this a typo or intended @AdrianValentinCiu ?

Comment (AdrianValentinCiu) [reply .github/workflows/notebooks.yml]: typo. fixed it now

**Review by liadomide — APPROVED** [review]

**Pull request merged**